lib/python/qmk/cli/fileformat.py

```diff
@@ -0,0 +1,13 @@
+"""Format files according to QMK's style.
+"""
+from milc import cli
+
+import subprocess
+
+
+@cli.subcommand("Format files according to QMK's style.")
```

Comment:
I'm thinking exposing this command to users might be confusing. 

```suggestion
@cli.subcommand("Format files according to QMK's style.", hidden=True)
```